src/components/CommentMessage/CommentMessage.stories.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+
+import { CommentMessage } from "./CommentMessage";
+import { comment } from "../../fixtures/comment";
+import { staffUser } from "../../fixtures/user";
```

Comment:
the abstraction originated because I needed a user, and saw one defined in `App.tsx`. 
I noticed that there are variations between staff and non staff users, so I think this abstraction is merited now.

Reply:
The problem though is it's only used in one place. So all you've achieved is to take some code from that file and put it in a different one. Now when someone comes to this file to learn about or fix a problem with this test they will need to context switch between two different files, which has a cognitive cost.

We should be slow to do things that increase this cost and only do them when there's benefit that makes the move worth it. That benefit could be if the file is getting excessively large requiring lots of scrolling or if the same code is shared in many places and, crucially, that you are confident that changes to that shared code should always be applied in all the places it is used.